config/application.rb

```diff
@@ -81,5 +81,10 @@ class Application < Rails::Application
     config.x.google_api_key = credentials.dig(Rails.env.to_sym, :google_api_key)
 
     config.x.healthcheck_ids = credentials.dig(Rails.env.to_sym, :healthcheck_ids)
+
+    config.x.noreply_email = ENV.fetch('NOREPLY_EMAIL', 'no-reply@csvalpha.com')
+    config.x.ict_email = ENV.fetch('ICT_EMAIL', 'ict@csvalpha.com')
+    config.x.privacy_email = ENV.fetch('PRIVACY_EMAIL', 'privacy@csvalpha.com')
+    config.x.mailbeheer_email = ENV.fetch('PRIVACY_EMAIL', 'mailbeheer@csvalpha.com')
```

Comment:
I'm not too familiar with this file. What does the config.x namespace represent?

Reply:
I dont completly understand your question. this lines looks for the constant in the env file if the find the they will use that value otherwise the will use the default value. adding a default value makes it so production doesnt break when merging. I used https://github.com/csvalpha/sofia/commit/0a75d04d369fce1e78a2925120818e42f0ef94de as a example

Reply:
I was on mobile when reviewing this the first time. The documentation I was looking for is this: https://guides.rubyonrails.org/configuring.html#config-x
So, to answer my own question:
Q: what is `config.x`?
A:`config.x` allows for custom configuration fields

Reply:
I now understand